app/bundles/EmailBundle/Tests/Helper/UrlMatcherTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * @copyright   2017 Mautic Contributors. All rights reserved
+ * @author      Mautic, Inc.
+ *
+ * @link        https://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\EmailBundle\Tests\Helper;
+
+use Mautic\EmailBundle\Helper\UrlMatcher;
+
+class UrlMatcherTest extends \PHPUnit_Framework_TestCase
```

Comment:
I would expect that test covers this URLs too:
`google.com` and `google.com/` - should match
`https://google.com` and http://google.com - ? Should match?
`https://google.com` and `google.com` - ?
`https://google.com` and `//google.com` - Should match?
`https://google.com` and `https://google.com/` - Should match
`https://google.com/hello` and `https://google.com/hello/` Should not match?

These tests will clarify function of the service. I am not able to tell, if it works or not without trying now. But I think that at least a few examples will fail. Or is function correct and my thoughts are bad?

Reply:
@Maxell92 I added support for these cases.